assets/src/containers/DashboardAppBar.js

```diff
@@ -57,7 +66,7 @@ function DashboardAppBar (props) {
           </IconButton>
           <Button>
             <Typography variant='h6' color='inherit' className={classes.grow}>
-              <Link to='/' className={classes.homeButton}>My Learning Analytics</Link>
+              <Link to={path} className={classes.homeButton}>My Learning Analytics</Link>
```

Comment:
I don't see the Course name is not displaying that is a real piece of information that is needed.

Reply:
The course name info is already available in the `/info` endpoint

Reply:
Just getting to this - how should the course name be displayed?

Reply:
Okay I've added the course name so that it shows up as "MY LEARNING ANALYTICS: Course Name"